cmake/Platform/Common/Install_common.cmake

```diff
@@ -167,18 +167,16 @@ function(ly_setup_target OUTPUT_CONFIGURED_TARGET ALIAS_TARGET_NAME absolute_tar
     endforeach()
     list(JOIN INCLUDE_DIRECTORIES_PLACEHOLDER "\n" INCLUDE_DIRECTORIES_PLACEHOLDER)
 
-    string(REPEAT " " 8 PLACEHOLDER_INDENT)
-    get_target_property(RUNTIME_DEPENDENCIES_PLACEHOLDER ${TARGET_NAME} MANUALLY_ADDED_DEPENDENCIES)
-    if(RUNTIME_DEPENDENCIES_PLACEHOLDER) # not found properties return the name of the variable with a "-NOTFOUND" at the end, here we set it to empty if not found
-        set(RUNTIME_DEPENDENCIES_PLACEHOLDER "${PLACEHOLDER_INDENT}${RUNTIME_DEPENDENCIES_PLACEHOLDER}")
-        list(JOIN RUNTIME_DEPENDENCIES_PLACEHOLDER "\n${PLACEHOLDER_INDENT}" RUNTIME_DEPENDENCIES_PLACEHOLDER)
-    else()
-        unset(RUNTIME_DEPENDENCIES_PLACEHOLDER)
-    endif()
-
     string(REPEAT " " 12 PLACEHOLDER_INDENT)
     get_property(interface_build_dependencies_props TARGET ${TARGET_NAME} PROPERTY LY_DELAYED_LINK)
     unset(INTERFACE_BUILD_DEPENDENCIES_PLACEHOLDER)
+    # We can have private build dependencies that contain directly or indirectly runtime dependencies.
```

Comment:
typo: `that contain directly or indirectly` -> `that contains direct or indirect`

Reply:
```suggestion
    # We can have private build dependencies that contains direct or indirect runtime dependencies.
```